src/components/editor/TextEditor/AddImage/ImageItem.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactComponent as Trash } from 'assets/svg/trash.svg';
+import { useRef, useState } from 'react';
+import useOnClickOutside from 'utils/hooks/useOnClickOutside';
+import styles from '../TextEditor.module.scss';
+
+interface Props {
+  value: string,
+  removeImage: (value: string) => void,
```

Comment:
removeImage 보단 onRemove 가 어떨까... 싶습니다. (이것도 별로 좋아보이진 않네요)

```suggestion
  onDelete : (value: string) => void,
```

Reply:
onDelete로 적용했습니다.